site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,250 @@
+
+##### A11y properties
+- `labels`: The Calendar widget has a number of internal descriptive labels, and this property allows their text to be customized
+
+___
+
+### Dialog
+
+The Dialog widget creates a controlled dialog element. Controlled focus is not yet implemented, but will be added along with the focus manager.
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close dialog."
+- `role`: The default role for the dialog container is `dialog`, but it can be changed to `alertdialog` through this property.
+
+___
+
+### SlidePane
+
+SlidePane is similar to dialog in that it creates a modal that overlays page content. It will need similar programmatic focus control, pending completion of the focus manager.
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close pane."
+
+___
+
+### TabController
+
+TabController creates a set of tab buttons that control the content of a container. The tab buttons use `role="tab"` and are associated with their content through `aria-controls` and `aria-labelledby`. Only one tab button is in the tab order at a time, and arrow keys are used to switch between tab buttons. This approach both directly follows the [ARIA Authoring Practice Guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/) and avoids polluting the tab order with a large number of tab stops in the case of a single TabController with many tabs.
+
+##### Keyboard Interaction
+- **Left arrow**: Moves to previous tab button, wrapping from first to last
+- **Right arrow**: Moves to next tab button, wrapping from last to first
+- **Down arrow**: Moves to next tab button only on vertically oriented tabs
+- **Up arrow**: Moves to previous tab button only on vertically oriented tabs
+- **Page Up**: Moves to first tab button
+- **Page Down**: Moves to last tab button
+- **Escape**: Closes focused tab button if it is closeable
+
+___
+
+### TitlePane
+
+TitlePane creates an accordion widget, which is functionally a button that expands a content container. Collapsed content is obscured with `aria-hidden`, and the button and content container are associated with `aria-controls` and `aria-labelledby`.
+
+##### A11y properties
+- `headingLevel`: Optionally customize the heading level of the button controlling the accordion.
+___
+
+This overview only touched on properties that primarily exist for accessibility-related reasons, but any design decision or property change will end up affecting accessibility in some way. For some of those, such as `invalid`, `disabled`, and `readOnly`, we handle setting ARIA attributes in the background in addition to toggling classes without any extra attention needed from the widget author. In other cases, such as `getResultLabel` in ComboBox or `renderMonthLabel` in Calendar, it is entirely up to the author to ensure the returned result includes clear, screen-reader-accessible text content.
```

Comment:
we handle  ... should make it clear if this is Dojo 2 for all widgets, or just built in widgets

Reply:
ComboBox -> `ComboBox` and Calendar -> `Calendar`